shared/gateway/gateway.go

```diff
@@ -1,29 +1,33 @@
-// Package gateway defines a gRPC gateway to serve HTTP-JSON
-// traffic as a proxy and forward it to a beacon node's gRPC service.
 package gateway
 
 import (
 	"context"
 	"fmt"
 	"net"
 	"net/http"
+	"path"
+	"strings"
 	"time"
 
 	gwruntime "github.com/grpc-ecosystem/grpc-gateway/runtime"
 	"github.com/pkg/errors"
 	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1_gateway"
 	pbrpc "github.com/prysmaticlabs/prysm/proto/beacon/rpc/v1_gateway"
+	pb "github.com/prysmaticlabs/prysm/proto/validator/accounts/v2_gateway"
 	"github.com/prysmaticlabs/prysm/shared"
+	"github.com/prysmaticlabs/prysm/validator/web"
+	"github.com/rs/cors"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/connectivity"
 	"google.golang.org/grpc/credentials"
 )
 
 var _ shared.Service = (*Gateway)(nil)
 
-// Gateway is the gRPC gateway to serve HTTP JSON traffic as a proxy and forward
-// it to the beacon-chain gRPC server.
+// Gateway is the gRPC gateway to serve HTTP JSON traffic as a
+// proxy and forward it to the gRPC server.
 type Gateway struct {
+	callerId                string
```

Comment:
Let's use an enumeration here. Example: https://github.com/prysmaticlabs/prysm/blob/648e3608429691e15ccbc0d62dc8089c4440c5e7/slasher/detection/attestations/types/types.go#L15

Reply:
done

Reply:
Can you please follow the same convention and use `iota`?

Reply:
Done